### ISSUE TITLE
Update jira-ruby dependency

### ISSUE THREAD

**Issue by osipxd** [RedMadRobot/fastlane-plugin-jira_release_notes#11]
`jira_release_notes` plugin depends on the old version of `jira-ruby` which is not compatible with `activesupport 7`.
Here is the issue: https://github.com/sumoheavy/jira-ruby/issues/385

https://github.com/RedMadRobot/fastlane-plugin-jira_release_notes/blob/0bd32de002e550896c1c541e1a2bfb6a921108ac/fastlane-plugin-jira_release_notes.gemspec#L24

This issue [was fixed](https://github.com/sumoheavy/jira-ruby/commit/5cf40699fb63c3ba3efe3f528aef5c2df25b1415) in jira-ruby 2.2.0. Is it possible to update the required version of jira-ruby?

**Comment by osipxd** [comment]
Example of lane failure:

```
  | 💥   | jira_release_notes | 0           |
  +------+--------------------+-------------+
  [13:55:55]: fastlane finished with errors
bundler: failed to load command: fastlane (/builds/android/vendor/ruby/3.0.0/bin/fastlane)
/builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:184:in `current_thread_backend': \e[31m[!] uninitialized constant ActiveSupport::XmlMini::IsolatedExecutionState\e[0m (NameError)
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:103:in `backend='
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:201:in `<module:ActiveSupport>'
from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/xml_mini.rb:11:in `<top (required)>'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/array/conversions.rb:3:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/array/conversions.rb:3:in `<top (required)>'
from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/duration.rb:3:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/duration.rb:3:in `<top (required)>'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/time/calculations.rb:3:in `require'
from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/time/calculations.rb:3:in `<top (required)>'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/string/conversions.rb:4:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/string/conversions.rb:4:in `<top (required)>'
from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/string.rb:3:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/activesupport-7.0.1/lib/active_support/core_ext/string.rb:3:in `<top (required)>'
	from /builds/android/vendor/ruby/3.0.0/gems/jira-ruby-1.1.3/lib/jira/base.rb:1:in `require'
from /builds/android/vendor/ruby/3.0.0/gems/jira-ruby-1.1.3/lib/jira/base.rb:1:in `<top (required)>'
	from /builds/android/vendor/ruby/3.0.0/gems/jira-ruby-1.1.3/lib/jira-ruby.rb:8:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/jira-ruby-1.1.3/lib/jira-ruby.rb:8:in `<top (required)>'
from /builds/android/vendor/ruby/3.0.0/gems/fastlane-plugin-jira_release_notes-0.6.0/lib/fastlane/plugin/jira_release_notes/actions/jira_release_notes_action.rb:6:in `require'
	from /builds/android/vendor/ruby/3.0.0/gems/fastlane-plugin-jira_release_notes-0.6.0/lib/fastlane/plugin/jira_release_notes/actions/jira_release_notes_action.rb:6:in `run'
```

**Comment by Alexander-Ignition** [comment]
@osipxd will this problem fix if you write `'jira-ruby', '~> 2.2.0.'` in `Gemfile` ?

**Comment by osipxd** [comment]
> @osipxd will this problem fix if you write `'jira-ruby', '~> 2.2.0.'` in `Gemfile` ?

Haven't tried it yet. I use this as a workaround:

```ruby
gem 'activesupport', '<7.0.0'
```

**Comment by osipxd** [comment]
Oops. Missclick

**Comment by Alexander-Ignition** [comment]
fixed in version 0.7.0 ✅

@osipxd please check 🙏

**Comment by osipxd** [comment]
Thanks! It works
